Dockerfile

```diff
@@ -1,30 +1,123 @@
+      description="Packer is a tool for creating identical machine images for multiple platforms from a single source configuration. Please submit issues to https://github.com/hashicorp/packer/issues"
+
+# This is the location of the releases.
+ENV HASHICORP_RELEASES=https://releases.hashicorp.com
+
+RUN set -eux && \
+    apk add --no-cache git bash wget openssl gnupg && \
+    gpg --keyserver keyserver.ubuntu.com --recv-keys C874011F0AB405110D02105534365D9472D7468F && \
+    mkdir -p /tmp/build && \
+    cd /tmp/build && \
+    apkArch="$(apk --print-arch)" && \
+    case "${apkArch}" in \
+        aarch64) packerArch='arm64' ;; \
+        armhf) packerArch='arm' ;; \
+        x86) packerArch='386' ;; \
+        x86_64) packerArch='amd64' ;; \
+        *) echo >&2 "error: unsupported architecture: ${apkArch} (see ${HASHICORP_RELEASES}/packer/${VERSION}/)" && exit 1 ;; \
+    esac && \
+    wget ${HASHICORP_RELEASES}/packer/${VERSION}/packer_${VERSION}_linux_${packerArch}.zip && \
+    wget ${HASHICORP_RELEASES}/packer/${VERSION}/packer_${VERSION}_SHA256SUMS && \
+    wget ${HASHICORP_RELEASES}/packer/${VERSION}/packer_${VERSION}_SHA256SUMS.sig && \
+    gpg --batch --verify packer_${VERSION}_SHA256SUMS.sig packer_${VERSION}_SHA256SUMS && \
+    grep packer_${VERSION}_linux_${packerArch}.zip packer_${VERSION}_SHA256SUMS | sha256sum -c && \
+    unzip -d /tmp/build packer_${VERSION}_linux_${packerArch}.zip && \
+    cp /tmp/build/packer /bin/packer && \
+    cd /tmp && \
+    rm -rf /tmp/build && \
+    gpgconf --kill all && \
+    apk del gnupg openssl && \
+    rm -rf /root/.gnupg && \
+    # Tiny smoke test to ensure the binary we downloaded runs
+    packer version
+
+ENTRYPOINT ["/bin/packer"]
+
+
+# Production docker `light` image, which `latest` points to.
+# Remember, this cannot be built locally.
+FROM docker.mirror.hashicorp.services/alpine:latest as release-default
+
+ARG VERSION
+ARG BIN_NAME
+
+# TARGETARCH and TARGETOS are set automatically when --platform is provided.
+ARG TARGETOS TARGETARCH
+
+LABEL name="Packer" \
+      maintainer="HashiCorp Packer Team <packer@hashicorp.com>" \
+      vendor="HashiCorp" \
+      version=$VERSION \
+      release=$VERSION \
+      summary="Packer is a tool for creating identical machine images for multiple platforms from a single source configuration." \
+      description="Packer is a tool for creating identical machine images for multiple platforms from a single source configuration. Please submit issues to https://github.com/hashicorp/packer/issues"
+
+RUN apk add --no-cache git bash wget openssl gnupg
+
+COPY dist/$TARGETOS/$TARGETARCH/$BIN_NAME /bin/
+
+ENTRYPOINT ["/bin/packer"]
+
+
+# Full development docker image
+# This image includes all source code found in this repository, 
+# and is also pushed to DockerHub under the `full` tag. 
+FROM docker.mirror.hashicorp.services/alpine:latest as release-full
```

Comment:
As this is the final target defined in the Dockerfile, it will be the default one that gets built when running `docker build`. In helloworld, we have a final line in the Dockerfile which is just `FROM dev` the result of that line is that the `dev` image becomes the default. Just something we might want to use here too for consistency?

Reply:
Ah, I didn't know that. I don't think we have that change included in any projects.